### PR TITLE
Use Net::IP::XS when available

### LINKED ISSUE #293: Make use of Net::IP::XS if available
For performance reasons, please use Net::IP::XS if it is available to the Perl version running.

There has been a serious performance degradation since introducing Net::IP instead of Net::IP::XS.

### PR DESCRIPTION
**Pull request by mattias-p**
Fixes #293. This cuts Travis time by approximately 30%.

### CODE REVIEW

File: lib/Zonemaster/Engine/Net/IP.pm [hunk 1]
```diff
@@ -3,45 +3,81 @@ package Zonemaster::Engine::Net::IP;
...
+    my $self = shift;
+    return $self->{_inner}->reverse_ip();
+}
+
+sub version {
+    my $self = shift;
+    return $self->{_inner}->version();
+}
+
+sub ip_is_ipv6 {
```
Comment (vlevigneron) [inline lib/Zonemaster/Engine/Net/IP.pm]: Could you explain why ip_is_ipv6 implementation is different from other methods ? Or am I missing something ?

Comment (mattias-p) [reply lib/Zonemaster/Engine/Net/IP.pm]: `ip_is_ipv6` is part of the procedural interface while the others are part of the OO interface.

Comment (mattias-p) [reply lib/Zonemaster/Engine/Net/IP.pm]: Do you think the documentation could be improved to make that clearer? If you like I could put the method documentations just above their respective implementations, and the POD headings would follow along. That way the code would be more clearly organized with different kinds of things under different kinds of headings as well.

Comment (vlevigneron) [reply lib/Zonemaster/Engine/Net/IP.pm]: No, it's fine for me, I should have read more closely Net::IP and Net::IP::XS documentations.

**Pull request merged**